src/esp/bindings/SensorBindings.cpp

```diff
@@ -125,12 +243,15 @@ void initSensorBindings(py::module& m) {
       .def_property_readonly("hfov", &VisualSensor::getFOV,
                              R"(The Field of View this VisualSensor uses.)")
       .def_property_readonly("framebuffer_size", &VisualSensor::framebufferSize)
-      .def_property_readonly("render_target", &VisualSensor::renderTarget);
+      .def_property_readonly("render_target", &VisualSensor::renderTarget)
+      .def(
+          "buffer", buffer<VisualSensor>,
+          R"(Get the torch tensor or numpy array buffer of the Sensor, initialize if it does not exist yet)");
 
   // === CameraSensor ====
   py::class_<CameraSensor, Magnum::SceneGraph::PyFeature<CameraSensor>,
              VisualSensor, Magnum::SceneGraph::PyFeatureHolder<CameraSensor>>(
-      m, "CameraSensor")
+      m, "CameraSensor", py::dynamic_attr())
```

Comment:
Why do we need dynamic_attrs here, still?

Reply:
Thanks for the catch!